File: Modul-01/python/PRAK104-2310817210009-MuhammadIbnuSina.py
# Input harga sepatu A dan B
harga_sepatu_A = float(input("Harga sepatu A adalah: "))
harga_sepatu_B = float(input("Harga sepatu B adalah: "))

# Menghitung harga sepatu A setelah mendapat diskon 13%
diskon_A = 13
harga_diskon_A = harga_sepatu_A * (100 - diskon_A) / 100

# Menghitung harga sepatu B setelah mendapat diskon 21%
diskon_B = 21
harga_diskon_B = harga_sepatu_B * (100 - diskon_B) / 100

# Menampilkan hasil
print(f"Sepatu A mendapat diskon {diskon_A}% sehingga harganya menjadi {int(harga_diskon_A)}")
print(f"Sepatu B mendapat diskon {diskon_B}% sehingga harganya menjadi {int(harga_diskon_B)}")

File: Modul-01/c/PRAK104-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    float hargaSepatuA, hargaSepatuB;
    
    printf("Harga sepatu A adalah: ");
    scanf("%f", &hargaSepatuA);
    
    printf("Harga sepatu B adalah: ");
    scanf("%f", &hargaSepatuB);
    
    // Menghitung harga setelah diskon 13% untuk sepatu A
    float diskonA = 0.13 * hargaSepatuA;
    float hargaDiskonA = hargaSepatuA - diskonA;
    
    // Menghitung harga setelah diskon 21% untuk sepatu B
    float diskonB = 0.21 * hargaSepatuB;
    float hargaDiskonB = hargaSepatuB - diskonB;
    
    printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi %.0f\n", hargaDiskonA);
    printf("Sepatu B mendapat diskon 21%% sehingga harganya menjadi %.0f\n", hargaDiskonB);
    
    return 0;
}
